### PR TITLE
#18064 fix to support temporal file id on the binary field to discove…

### PR DESCRIPTION
**Pull request by jdotcms**
The mechanism to auto discovery the dotAsset content type was successfully able to work with a file sent as part of the multipart body.
However, when the binary field was sent as part of the body as a temporal file id it was not working.
The fix allows the mechanism to support also this variant

### CODE REVIEW

File: dotCMS/src/main/java/com/dotcms/rest/ContentHelper.java [hunk 1]
```diff
@@ -23,6 +23,8 @@
 
 import javax.servlet.http.HttpServletRequest;
 import java.io.File;
+import java.util.ArrayList;
```
Comment (dev-dotcms) [inline dotCMS/src/main/java/com/dotcms/rest/ContentHelper.java]: Codacy Issue found: [Avoid unused imports such as 'java.util.ArrayList'](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5093431)

**Review by nollymar — COMMENTED** [review]
it would be great to add a few ITs

**Review by erickgonzalez — APPROVED** [review]

File: dotCMS/src/integration-test/java/com/dotcms/contenttype/business/DotAssetBaseTypeToContentTypeStrategyImplTest.java [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+        APILocator.getContentTypeFieldAPI().save(binaryField, APILocator.systemUser());
+        APILocator.getContentTypeFieldAPI().save(allowFileTypes, APILocator.systemUser());
+
+        final Optional<BaseTypeToContentTypeStrategy> baseTypeToContentTypeStrategy =
+                BaseTypeToContentTypeStrategyResolver.getInstance().get(BaseContentType.DOTASSET);
+
+        Assert.assertTrue(baseTypeToContentTypeStrategy.isPresent());
+        MockHeaderRequest request = new MockHeaderRequest(
+                (
+                        new MockSessionRequest(new MockAttributeRequest(new MockHttpRequest("localhost", "/").request()).request())
```
Comment (dev-dotcms) [inline dotCMS/src/integration-test/java/com/dotcms/contenttype/business/DotAssetBaseTypeToContentTypeStrategyImplTest.java]: Codacy Issue found: [Useless parentheses.](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5093431)

File: dotCMS/src/integration-test/java/com/dotcms/contenttype/business/DotAssetBaseTypeToContentTypeStrategyImplTest.java [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+
+        // creates a dotAsset for text files
+        final String variable = "testDotAsset" + System.currentTimeMillis();
+        final ContentType dotAssetContentType = APILocator.getContentTypeAPI(APILocator.systemUser()).
+                save(ContentTypeBuilder.builder(DotAssetContentType.class).folder(
+                FolderAPI.SYSTEM_FOLDER).host(Host.SYSTEM_HOST).name(variable)
+                .owner(APILocator.systemUser().getUserId()).build());
+
+        final Map<String, Field> fieldMap = dotAssetContentType.fieldMap();
+        com.dotcms.contenttype.model.field.Field binaryField           = fieldMap.get(DotAssetContentType.ASSET_FIELD_VAR);
```
Comment (dev-dotcms) [inline dotCMS/src/integration-test/java/com/dotcms/contenttype/business/DotAssetBaseTypeToContentTypeStrategyImplTest.java]: Codacy Issue found: [Unnecessary use of fully qualified name 'com.dotcms.contenttype.model.field.Field' due to existing import 'com.dotcms.contenttype.model.field.Field'](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5093431)

File: dotCMS/src/integration-test/java/com/dotcms/contenttype/business/DotAssetBaseTypeToContentTypeStrategyImplTest.java [hunk 4]
```diff
@@ -0,0 +1,118 @@
...
+import com.dotcms.mock.request.MockSessionRequest;
+import com.dotcms.rest.api.v1.temp.DotTempFile;
+import com.dotcms.util.CollectionsUtils;
+import com.dotcms.util.IntegrationTestInitService;
+import com.dotmarketing.beans.Host;
+import com.dotmarketing.business.APILocator;
+import com.dotmarketing.portlets.folders.business.FolderAPI;
+import com.dotmarketing.util.FileUtil;
+import com.dotmarketing.util.UUIDGenerator;
+import com.liferay.portal.model.User;
```
Comment (dev-dotcms) [inline dotCMS/src/integration-test/java/com/dotcms/contenttype/business/DotAssetBaseTypeToContentTypeStrategyImplTest.java]: Codacy Issue found: [Avoid unused imports such as 'com.liferay.portal.model.User'](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5093431)

File: dotCMS/src/integration-test/java/com/dotcms/contenttype/business/DotAssetBaseTypeToContentTypeStrategyImplTest.java [hunk 5]
```diff
@@ -0,0 +1,118 @@
...
+import java.io.File;
+import java.io.FileWriter;
+import java.util.Arrays;
+import java.util.Map;
+import java.util.Optional;
+
+public class DotAssetBaseTypeToContentTypeStrategyImplTest  extends IntegrationTestBase {
+
+    @BeforeClass
+    public static void prepare() throws Exception {
```
Comment (dev-dotcms) [inline dotCMS/src/integration-test/java/com/dotcms/contenttype/business/DotAssetBaseTypeToContentTypeStrategyImplTest.java]: Codacy Issue found: [A method/constructor should not explicitly throw java.lang.Exception](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5093431)

File: dotCMS/src/integration-test/java/com/dotcms/contenttype/business/DotAssetBaseTypeToContentTypeStrategyImplTest.java (hunk 3 above)
Comment (dev-dotcms) [inline dotCMS/src/integration-test/java/com/dotcms/contenttype/business/DotAssetBaseTypeToContentTypeStrategyImplTest.java]: Codacy Issue found: [Local variable 'binaryField' could be declared final](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5093431)

File: dotCMS/src/main/java/com/dotcms/rest/ContentHelper.java (hunk 1 above)
Comment (dev-dotcms) [inline dotCMS/src/main/java/com/dotcms/rest/ContentHelper.java]: Codacy Issue found: [Avoid unused imports such as 'java.util.ArrayList'](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5093431)

File: dotCMS/src/integration-test/java/com/dotcms/contenttype/business/DotAssetBaseTypeToContentTypeStrategyImplTest.java [hunk 6]
```diff
@@ -0,0 +1,118 @@
...
+
+        APILocator.getContentTypeAPI(APILocator.systemUser()).save(dotAssetContentType);
+        APILocator.getContentTypeFieldAPI().save(binaryField, APILocator.systemUser());
+        APILocator.getContentTypeFieldAPI().save(allowFileTypes, APILocator.systemUser());
+
+        final Optional<BaseTypeToContentTypeStrategy> baseTypeToContentTypeStrategy =
+                BaseTypeToContentTypeStrategyResolver.getInstance().get(BaseContentType.DOTASSET);
+
+        Assert.assertTrue(baseTypeToContentTypeStrategy.isPresent());
+        MockHeaderRequest request = new MockHeaderRequest(
```
Comment (dev-dotcms) [inline dotCMS/src/integration-test/java/com/dotcms/contenttype/business/DotAssetBaseTypeToContentTypeStrategyImplTest.java]: Codacy Issue found: [Local variable 'request' could be declared final](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5093431)

File: dotCMS/src/integration-test/java/com/dotcms/contenttype/business/DotAssetBaseTypeToContentTypeStrategyImplTest.java [hunk 7]
```diff
@@ -0,0 +1,118 @@
...
+import com.dotcms.rest.api.v1.temp.DotTempFile;
+import com.dotcms.util.CollectionsUtils;
+import com.dotcms.util.IntegrationTestInitService;
+import com.dotmarketing.beans.Host;
+import com.dotmarketing.business.APILocator;
+import com.dotmarketing.portlets.folders.business.FolderAPI;
+import com.dotmarketing.util.FileUtil;
+import com.dotmarketing.util.UUIDGenerator;
+import com.liferay.portal.model.User;
+import com.liferay.portal.util.PortalUtil;
```
Comment (dev-dotcms) [inline dotCMS/src/integration-test/java/com/dotcms/contenttype/business/DotAssetBaseTypeToContentTypeStrategyImplTest.java]: Codacy Issue found: [Avoid unused imports such as 'com.liferay.portal.util.PortalUtil'](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5093431)

File: dotCMS/src/integration-test/java/com/dotcms/contenttype/business/DotAssetBaseTypeToContentTypeStrategyImplTest.java [hunk 8]
```diff
@@ -0,0 +1,118 @@
...
+
+        request.setHeader("Authorization", "Basic " + new String(Base64.encode("admin@dotcms.com:admin".getBytes())));
+        request.setHeader("User-Agent", "Fake-Agent");
+        request.setHeader("Host", "localhost");
+        request.setHeader("Origin", "localhost");
+        request.setAttribute(WebKeys.USER, APILocator.systemUser());
+
+        final File file = FileUtil.createTemporalFile("test", "txt");
+        final String content = "This is a test temporal file";
+        try (final FileWriter fileWriter = new FileWriter(file)) {
```
Comment (dev-dotcms) [inline dotCMS/src/integration-test/java/com/dotcms/contenttype/business/DotAssetBaseTypeToContentTypeStrategyImplTest.java]: Codacy Issue found: [Unnecessary modifier 'final' on resource specification 'fileWriter': resource specifications are implicitly final](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5093431)

Comment (dev-dotcms) [inline dotCMS/src/integration-test/java/com/dotcms/contenttype/business/DotAssetBaseTypeToContentTypeStrategyImplTest.java]: Codacy Issue found: [Avoid instantiating FileInputStream, FileOutputStream, FileReader, or FileWriter](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5093431)

Comment (dev-dotcms) [inline dotCMS/src/integration-test/java/com/dotcms/contenttype/business/DotAssetBaseTypeToContentTypeStrategyImplTest.java]: Codacy Issue found: [Unnecessary modifier 'final' on resource specification 'fileWriter': resource specifications are implicitly final](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5093431)

File: dotCMS/src/main/java/com/dotcms/rest/ContentHelper.java (hunk 1 above)
Comment (dev-dotcms) [inline dotCMS/src/main/java/com/dotcms/rest/ContentHelper.java]: Codacy Issue found: [Avoid unused imports such as 'java.util.ArrayList'](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5093431)

File: dotCMS/src/integration-test/java/com/dotcms/contenttype/business/DotAssetBaseTypeToContentTypeStrategyImplTest.java (hunk 3 above)
Comment (dev-dotcms) [inline dotCMS/src/integration-test/java/com/dotcms/contenttype/business/DotAssetBaseTypeToContentTypeStrategyImplTest.java]: Codacy Issue found: [Unnecessary use of fully qualified name 'com.dotcms.contenttype.model.field.Field' due to existing import 'com.dotcms.contenttype.model.field.Field'](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5093431)

File: dotCMS/src/integration-test/java/com/dotcms/contenttype/business/DotAssetBaseTypeToContentTypeStrategyImplTest.java (hunk 2 above)
Comment (dev-dotcms) [inline dotCMS/src/integration-test/java/com/dotcms/contenttype/business/DotAssetBaseTypeToContentTypeStrategyImplTest.java]: Codacy Issue found: [Useless parentheses.](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5093431)

File: dotCMS/src/integration-test/java/com/dotcms/contenttype/business/DotAssetBaseTypeToContentTypeStrategyImplTest.java (hunk 4 above)
Comment (dev-dotcms) [inline dotCMS/src/integration-test/java/com/dotcms/contenttype/business/DotAssetBaseTypeToContentTypeStrategyImplTest.java]: Codacy Issue found: [Avoid unused imports such as 'com.liferay.portal.model.User'](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5093431)

File: dotCMS/src/integration-test/java/com/dotcms/contenttype/business/DotAssetBaseTypeToContentTypeStrategyImplTest.java (hunk 7 above)
Comment (dev-dotcms) [inline dotCMS/src/integration-test/java/com/dotcms/contenttype/business/DotAssetBaseTypeToContentTypeStrategyImplTest.java]: Codacy Issue found: [Avoid unused imports such as 'com.liferay.portal.util.PortalUtil'](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5093431)

File: dotCMS/src/integration-test/java/com/dotcms/contenttype/business/DotAssetBaseTypeToContentTypeStrategyImplTest.java (hunk 5 above)
Comment (dev-dotcms) [inline dotCMS/src/integration-test/java/com/dotcms/contenttype/business/DotAssetBaseTypeToContentTypeStrategyImplTest.java]: Codacy Issue found: [A method/constructor should not explicitly throw java.lang.Exception](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5093431)

File: dotCMS/src/integration-test/java/com/dotcms/contenttype/business/DotAssetBaseTypeToContentTypeStrategyImplTest.java (hunk 6 above)
Comment (dev-dotcms) [inline dotCMS/src/integration-test/java/com/dotcms/contenttype/business/DotAssetBaseTypeToContentTypeStrategyImplTest.java]: Codacy Issue found: [Local variable 'request' could be declared final](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5093431)

File: dotCMS/src/integration-test/java/com/dotcms/contenttype/business/DotAssetBaseTypeToContentTypeStrategyImplTest.java (hunk 3 above)
Comment (dev-dotcms) [inline dotCMS/src/integration-test/java/com/dotcms/contenttype/business/DotAssetBaseTypeToContentTypeStrategyImplTest.java]: Codacy Issue found: [Local variable 'binaryField' could be declared final](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5093431)

File: dotCMS/src/integration-test/java/com/dotcms/contenttype/business/DotAssetBaseTypeToContentTypeStrategyImplTest.java (hunk 8 above)
Comment (dev-dotcms) [inline dotCMS/src/integration-test/java/com/dotcms/contenttype/business/DotAssetBaseTypeToContentTypeStrategyImplTest.java]: Codacy Issue found: [Avoid instantiating FileInputStream, FileOutputStream, FileReader, or FileWriter](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5093431)

File: dotCMS/src/integration-test/java/com/dotcms/contenttype/business/DotAssetBaseTypeToContentTypeStrategyImplTest.java (hunk 5 above)
Comment (dev-dotcms) [inline dotCMS/src/integration-test/java/com/dotcms/contenttype/business/DotAssetBaseTypeToContentTypeStrategyImplTest.java]: Codacy Issue found: [A method/constructor should not explicitly throw java.lang.Exception](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5093431)

File: dotCMS/src/integration-test/java/com/dotcms/contenttype/business/DotAssetBaseTypeToContentTypeStrategyImplTest.java (hunk 3 above)
Comment (dev-dotcms) [inline dotCMS/src/integration-test/java/com/dotcms/contenttype/business/DotAssetBaseTypeToContentTypeStrategyImplTest.java]: Codacy Issue found: [Unnecessary use of fully qualified name 'com.dotcms.contenttype.model.field.Field' due to existing import 'com.dotcms.contenttype.model.field.Field'](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5093431)

Comment (dev-dotcms) [inline dotCMS/src/integration-test/java/com/dotcms/contenttype/business/DotAssetBaseTypeToContentTypeStrategyImplTest.java]: Codacy Issue found: [Local variable 'binaryField' could be declared final](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5093431)

File: dotCMS/src/integration-test/java/com/dotcms/contenttype/business/DotAssetBaseTypeToContentTypeStrategyImplTest.java (hunk 8 above)
Comment (dev-dotcms) [inline dotCMS/src/integration-test/java/com/dotcms/contenttype/business/DotAssetBaseTypeToContentTypeStrategyImplTest.java]: Codacy Issue found: [Unnecessary modifier 'final' on resource specification 'fileWriter': resource specifications are implicitly final](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5093431)

File: dotCMS/src/integration-test/java/com/dotcms/contenttype/business/DotAssetBaseTypeToContentTypeStrategyImplTest.java (hunk 4 above)
Comment (dev-dotcms) [inline dotCMS/src/integration-test/java/com/dotcms/contenttype/business/DotAssetBaseTypeToContentTypeStrategyImplTest.java]: Codacy Issue found: [Avoid unused imports such as 'com.liferay.portal.model.User'](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5093431)

File: dotCMS/src/main/java/com/dotcms/rest/ContentHelper.java (hunk 1 above)
Comment (dev-dotcms) [inline dotCMS/src/main/java/com/dotcms/rest/ContentHelper.java]: Codacy Issue found: [Avoid unused imports such as 'java.util.ArrayList'](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5093431)

File: dotCMS/src/integration-test/java/com/dotcms/contenttype/business/DotAssetBaseTypeToContentTypeStrategyImplTest.java (hunk 7 above)
Comment (dev-dotcms) [inline dotCMS/src/integration-test/java/com/dotcms/contenttype/business/DotAssetBaseTypeToContentTypeStrategyImplTest.java]: Codacy Issue found: [Avoid unused imports such as 'com.liferay.portal.util.PortalUtil'](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5093431)

File: dotCMS/src/integration-test/java/com/dotcms/contenttype/business/DotAssetBaseTypeToContentTypeStrategyImplTest.java (hunk 8 above)
Comment (dev-dotcms) [inline dotCMS/src/integration-test/java/com/dotcms/contenttype/business/DotAssetBaseTypeToContentTypeStrategyImplTest.java]: Codacy Issue found: [Avoid instantiating FileInputStream, FileOutputStream, FileReader, or FileWriter](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5093431)

File: dotCMS/src/integration-test/java/com/dotcms/contenttype/business/DotAssetBaseTypeToContentTypeStrategyImplTest.java (hunk 6 above)
Comment (dev-dotcms) [inline dotCMS/src/integration-test/java/com/dotcms/contenttype/business/DotAssetBaseTypeToContentTypeStrategyImplTest.java]: Codacy Issue found: [Local variable 'request' could be declared final](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5093431)

File: dotCMS/src/integration-test/java/com/dotcms/contenttype/business/DotAssetBaseTypeToContentTypeStrategyImplTest.java (hunk 2 above)
Comment (dev-dotcms) [inline dotCMS/src/integration-test/java/com/dotcms/contenttype/business/DotAssetBaseTypeToContentTypeStrategyImplTest.java]: Codacy Issue found: [Useless parentheses.](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5093431)

**Review by nollymar — APPROVED** [review]

File: dotCMS/src/integration-test/java/com/dotcms/contenttype/business/DotAssetBaseTypeToContentTypeStrategyImplTest.java (hunk 4 above)
Comment (dev-dotcms) [inline dotCMS/src/integration-test/java/com/dotcms/contenttype/business/DotAssetBaseTypeToContentTypeStrategyImplTest.java:24]: Codacy Issue found: [Avoid unused imports such as 'com.liferay.portal.model.User'](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5093431)

File: dotCMS/src/integration-test/java/com/dotcms/contenttype/business/DotAssetBaseTypeToContentTypeStrategyImplTest.java (hunk 8 above)
Comment (dev-dotcms) [inline dotCMS/src/integration-test/java/com/dotcms/contenttype/business/DotAssetBaseTypeToContentTypeStrategyImplTest.java:92]: Codacy Issue found: [Avoid instantiating FileInputStream, FileOutputStream, FileReader, or FileWriter](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5093431)

File: dotCMS/src/integration-test/java/com/dotcms/contenttype/business/DotAssetBaseTypeToContentTypeStrategyImplTest.java (hunk 7 above)
Comment (dev-dotcms) [inline dotCMS/src/integration-test/java/com/dotcms/contenttype/business/DotAssetBaseTypeToContentTypeStrategyImplTest.java:25]: Codacy Issue found: [Avoid unused imports such as 'com.liferay.portal.util.PortalUtil'](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5093431)

File: dotCMS/src/integration-test/java/com/dotcms/contenttype/business/DotAssetBaseTypeToContentTypeStrategyImplTest.java (hunk 6 above)
Comment (dev-dotcms) [inline dotCMS/src/integration-test/java/com/dotcms/contenttype/business/DotAssetBaseTypeToContentTypeStrategyImplTest.java:78]: Codacy Issue found: [Local variable 'request' could be declared final](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5093431)

File: dotCMS/src/integration-test/java/com/dotcms/contenttype/business/DotAssetBaseTypeToContentTypeStrategyImplTest.java (hunk 3 above)
Comment (dev-dotcms) [inline dotCMS/src/integration-test/java/com/dotcms/contenttype/business/DotAssetBaseTypeToContentTypeStrategyImplTest.java:65]: Codacy Issue found: [Local variable 'binaryField' could be declared final](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5093431)

File: dotCMS/src/integration-test/java/com/dotcms/contenttype/business/DotAssetBaseTypeToContentTypeStrategyImplTest.java (hunk 8 above)
Comment (dev-dotcms) [inline dotCMS/src/integration-test/java/com/dotcms/contenttype/business/DotAssetBaseTypeToContentTypeStrategyImplTest.java:92]: Codacy Issue found: [Unnecessary modifier 'final' on resource specification 'fileWriter': resource specifications are implicitly final](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5093431)

File: dotCMS/src/integration-test/java/com/dotcms/contenttype/business/DotAssetBaseTypeToContentTypeStrategyImplTest.java (hunk 2 above)
Comment (dev-dotcms) [inline dotCMS/src/integration-test/java/com/dotcms/contenttype/business/DotAssetBaseTypeToContentTypeStrategyImplTest.java:80]: Codacy Issue found: [Useless parentheses.](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5093431)

File: dotCMS/src/integration-test/java/com/dotcms/contenttype/business/DotAssetBaseTypeToContentTypeStrategyImplTest.java (hunk 5 above)
Comment (dev-dotcms) [inline dotCMS/src/integration-test/java/com/dotcms/contenttype/business/DotAssetBaseTypeToContentTypeStrategyImplTest.java:41]: Codacy Issue found: [A method/constructor should not explicitly throw java.lang.Exception](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5093431)

File: dotCMS/src/integration-test/java/com/dotcms/contenttype/business/DotAssetBaseTypeToContentTypeStrategyImplTest.java (hunk 3 above)
Comment (dev-dotcms) [inline dotCMS/src/integration-test/java/com/dotcms/contenttype/business/DotAssetBaseTypeToContentTypeStrategyImplTest.java:65]: Codacy Issue found: [Unnecessary use of fully qualified name 'com.dotcms.contenttype.model.field.Field' due to existing import 'com.dotcms.contenttype.model.field.Field'](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5093431)

File: dotCMS/src/main/java/com/dotcms/rest/ContentHelper.java (hunk 1 above)
Comment (dev-dotcms) [inline dotCMS/src/main/java/com/dotcms/rest/ContentHelper.java:26]: Codacy Issue found: [Avoid unused imports such as 'java.util.ArrayList'](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5093431)

**Pull request merged**